docs/content/utilities/sqs_large_message_handling.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: SQS Large Message Handling
+description: Utility
+---
+
+The large message handling utility handles SQS messages which have had their payloads
+offloaded to S3 due to them being larger than the SQS maximum.
+
+The utility automatically retrieves messages which have been offloaded to S3 using the
+[amazon-sqs-java-extended-client-lib](https://github.com/awslabs/amazon-sqs-java-extended-client-lib)
+client library. Once the message payloads have been processed successful the
+utility can delete the message payloads from S3.
+
+This utility is compatible with versions *1.1.0+* of amazon-sqs-java-extended-client-lib.</p>
+
+```xml
+<dependency>
+    <groupId>com.amazonaws</groupId>
+    <artifactId>amazon-sqs-java-extended-client-lib</artifactId>
+    <version>1.1.0</version>
+</dependency>
+```
+
+## Install
+
+To install this utility, add the following dependency to your project.
+
+```xml
+<dependency>
+    <groupId>software.amazon.lambda</groupId>
+    <artifactId>powertools-sqs</artifactId>
+    <version>x.x.x</version>
```

Comment:
may be update with version latest version here ? or can be done as part of release